OpenSim/Tools/InverseKinematicsTool.cpp

```diff
@@ -283,6 +283,9 @@ bool InverseKinematicsTool::run()
 
         cout<<"Running tool "<<getName()<<".\n";
 
+		// Get the trial name to label data written to files
+		string TrialName = getName();
```

Comment:
Variable names should start with a lower case letter; change this to `trialName`